packages/cxl-ui/scss/global/cxl-adeft-accordion.scss

```diff
@@ -1,56 +1,50 @@
 @use "~@conversionxl/cxl-lumo-styles/scss/mq";
 @use "../mixins";
 
-$bg-color: #0000ff;
 $standard-margin: 16px;
 $medium-margin: 24px;
 $large-margin: 32px;
 
 cxl-adeft-accordion {
   vaadin-vertical-layout {
     display: flex;
-    border-top: 1px solid $bg-color;
-    border-left: 4px solid $bg-color;
   }
-  --lumo-secondary-text-color: #000;
+
   --lumo-line-height-m: 20px;
   --lumo-line-height-xs: 20px;
-  --lumo-primary-color: $bg-color;
-  --lumo-contrast-20pct: $bg-color;
   margin-bottom: $standard-margin;
 
-  &.widened {
+  &[widened] {
     margin-right: -#{$standard-margin - 1};
     margin-left: -#{$standard-margin - 1};
   }
 
   @media #{mq.$medium} {
-    &.widened {
+    &[widened] {
       margin-right: 0;
       margin-left: 0;
     }
   }
 
-  .itemContent {
+  .item-content {
+    padding: $standard-margin $standard-margin / 2;
     letter-spacing: 0.25px;
-    padding: $standard-margin $standard-margin/2;
   }
 
-  .accordionSummary {
+  .accordion-summary {
```

Comment:
Why invent a new class instead of `slot[name="summary"]`?

Reply:
> Why invent a new class instead of `slot[name="summary"]`?
fixed in last commit